### ISSUE TITLE
Update .gitpod.yml to include mdb image and env variables

### ISSUE THREAD

**Issue by PeopleMakeCulture** [microbiomedata/nmdc-runtime#362]

**Comment by PeopleMakeCulture** [comment]
NOTE: I believe `make up-dev` needs to be moved out of `init` and into `command` since `init` is executed once and expected to terminate 

```yml
tasks:
  - name: Start Dev on Fresh Gitpod
    before: cp .env.example .env
    command: make up-dev
```

see: https://www.gitpod.io/docs/references/gitpod-yml#tasksninit

**Comment by dwinston** [comment]
closed via ace24868d8f770c4833b2d2962aa6769d9a3ce03
